docs/enhancements/0000-break-commands-into-plugins.md

```diff
@@ -0,0 +1,132 @@
+# Breaking toolbelt commands into plugins
+
+Remove all the commands, transform them into plugins and set only the necessary ones as default when you install the VTEX CLI.
+
+## Motivation
+
+VTEX CLI is the first contact that we have with the VTEX IO products, so we need to have a clean and simple CLI that works pretty well.
+
+Nowadays we have a CLI that, by default, comes with a **LOT** of commands that probably most of them will never be used by the same user.
+
+![](https://i.imgur.com/2GIq2Ep.png)
+
+The idea is to remove this commands that are not necessary to come by default, i.e., `vtex lighthouse` and `vtex test`. Eventually the user will need to use some of this extra commands, so just import to your own CLI.
+
+We can also have a library with all the possible commands that the user can import such as [Adobe I/O CLI](https://github.com/adobe/aio-cli).
+
+![](https://i.imgur.com/TXS0MmO.png)
+
+With this, the user will have a better experience with a new and clean CLI.
+
+## Explanation
+
+This new change will be made by `plugins`, breaking all the commands and transform them into plugins, now we are going to add only necessary `plugins` as default. Some documentation about plugins was made by a study and is registered [here](https://github.com/VerasThiago/CLIBenchmark/blob/master/Oclif/docs/ABOUTME.md).
+
+## Internal details
+
+### Toolbelt Plugins
+
+Huge amount of toolbelt commands share the same piece of code, in order that, toolbelt plugins will need to use it as well. So what do toolbelt-plugins will need?
```

Comment:
```suggestion

Toolbelt commands share a lot of common code, which is a situation that will happen too when the commands are broken into plugins. Because of this we have to create a way to share this common code across plugins, but first we have to determine what we want to share across them:
```